### DISCUSSION TITLE
Adding a filter does not reset the paging of the search results, resulting in a 'nothing found page'

### DISCUSSION THREAD

**Discussion by joerg1985** [community/community#125571] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Adding a filter to the code search will not reset the paging,
 
e.g. you just searched `isdisplayed` inside the selenium repo and navigated to the last page: 
https://github.com/search?q=repo%3ASeleniumHQ%2Fselenium+isdisplayed&type=code&p=3

Now add a filter to language 'JavaScript' via the button
![image](https://github.com/community/community/assets/16140691/12bc64e0-898d-4184-b45d-0ae59ca59d79)

You will end on the error page:
https://github.com/search?q=repo%3ASeleniumHQ%2Fselenium+isdisplayed+language%3AJavaScript&type=code&p=3&l=JavaScript
![image](https://github.com/community/community/assets/16140691/67f55c61-6ba0-4d55-917b-ce782ef62f51)

**Comment by LiteBrite82** [comment, marked as answer]
@joerg1985 

Thanks for the report! The engineering team is aware of the issue: https://github.com/orgs/community/discussions/123124
